api/v2/api.js

```diff
@@ -134,7 +142,23 @@ router.get('/jumble', async (req, res) => {
 
 });
 
-router.get('')
+router.get('/grumble', async(req, res) => {
+
+  let limit = parseInt(req.query.lat) || 10;
```

Comment:
req.query.lat should be req.query.limit right?

Reply:
updated in new commit